src/menu/ide_menu.rs

```diff
@@ -680,12 +699,18 @@ impl Menu for IdeMenu {
             // Also, by replacing the new line character with a space, the insert
             // position is maintain in the line buffer.
             let trimmed_buffer = editor.get_buffer().replace("\r\n", "  ").replace('\n', " ");
-            completer.complete(
+            completer.complete_with_base_ranges(
```

Comment:
The expanded example with the added ability to throw some new lines in there will test this part of the code better